src/draw/sw/lv_draw_sw.c

```diff
@@ -537,38 +535,38 @@ static void execute_drawing(lv_draw_sw_unit_t * u)
     lv_draw_task_t * t = u->task_act;
     switch(t->type) {
         case LV_DRAW_TASK_TYPE_FILL:
-            lv_draw_sw_fill((lv_draw_unit_t *)u, t->draw_dsc, &t->area);
+            lv_draw_sw_fill(t, t->draw_dsc, &t->area);
```

Comment:
why not removing all the other arguments and simply use just `t`?

Reply:
> why not removing all the other arguments and simply use just `t`?

The reach of this change is much bigger and although doable I prefer to not mess with it in this patch.